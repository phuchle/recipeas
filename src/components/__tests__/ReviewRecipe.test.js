import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ReviewRecipe from '../ReviewRecipe';

const props = {
  buttonStyle: {},
  addRecipe: jest.fn(),
  tempRecipe: {},
  clearTempRecipe: jest.fn()
};

describe('ReviewRecipe', () => {
  test('renders without crashing', () => {
    const component = shallow(<ReviewRecipe {...props} />);
    const tree = toJson(component);

    expect(tree).toMatchSnapshot();
  });
});
