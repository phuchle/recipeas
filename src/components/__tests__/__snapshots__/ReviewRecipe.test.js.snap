// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewRecipe renders without crashing 1`] = `
<div>
  <h4>
    Review Recipe
  </h4>
  <Connect(ModifyTitle)
    nextButton={false}
  />
  <Connect(ModifyIngredients)
    nextButton={false}
    plusButton={false}
  />
  <Link
    replace={false}
    to="/"
  >
    <Button
      active={false}
      block={true}
      bsClass="btn"
      bsSize="large"
      bsStyle="primary"
      disabled={false}
      onClick={[Function]}
      style={Object {}}
    >
      Submit
    </Button>
  </Link>
  <Link
    replace={false}
    to="/modify-ingredients"
  >
    <Button
      active={false}
      block={true}
      bsClass="btn"
      bsSize="large"
      bsStyle="default"
      disabled={false}
    >
      Back
    </Button>
  </Link>
</div>
`;
